engines/dfc_provider/app/services/authorization_control.rb

```diff
@@ -30,6 +30,10 @@ def oidc_user
     find_ofn_user(decode_token) if access_token
   end
 
+  def ofn_api_user
+    Spree::User.find_by(spree_api_key: ofn_api_token) if ofn_api_token.present?
```

Comment:
I see that we do the same here for v0 and v1 APIs, so I'd like to share the common code. But it's so simple it seems pretty safe for now.